pkg/platform/aks/aks.go

```diff
@@ -83,7 +83,7 @@ const (
 	subscriptionIDEnv = "LOKOMOTIVE_AKS_SUBSCRIPTION_ID"
 	tenantIDEnv       = "LOKOMOTIVE_AKS_TENANT_ID"
 
-	kubernetesVersion = "1.18.10"
+	kubernetesVersion = "1.18.17"
```

Comment:
should this change be here?

Reply:
It has been merged as part of #1466, so it won't be part of merge commit. Rebasing will remove it from view here.